lib/utils.ts

```diff
@@ -16,3 +16,13 @@ export const envVarsPresent = () => {
 
   return !!gigsProject && !!gigsAPIKey
 }
+
+const privateEnvVars = {
+  APP_ENV: process.env.APP_ENV,
+  GIGS_PROJECT: process.env.GIGS_PROJECT,
```

Comment:
aren't these public vars?

Reply:
Well, technically yes, I just stole and modified a little the function and  variable from portals 😅 I can rename them to public for clarity 👍 😄

Reply:
I'm indifferent. Don't worry about it 👍